cache/default.go

```diff
@@ -0,0 +1,76 @@
+package cache
+
+import (
+	"fmt"
+	"time"
+)
+
+// NewDefaultCache returns an instance of the default cache.
+func NewDefaultCache() *DefaultCache {
+	return &DefaultCache{items: make(map[string]*Item)}
```

Comment:
Lets have a goroutine here calling ExpireKeys so the client doesn't have to handle that.